modules/nw-operator-cr.adoc

```diff
@@ -206,7 +206,7 @@ endif::operator[]
 
 |`gatewayConfig`
 |`object`
-|Optional: Specify a configuration object for customizing how egress traffic is sent to the node gateway.
+|Optional: Specify a configuration object for customizing how egress traffic is sent to the node gateway. Accepted values are `Shared` and `Local`. The default value is `Shared`. In the default setting, the Open vSwitch (OVS) outputs traffic directly to the node IP interface, In the `Local` setting, it traverses the host network; consequently, it gets applied to the routing table of the host.
```

Comment:
@JoeAldinger Do you want to say possible values are "Shared" and "Local" instead of accepted? It is not we are setting these as values but they are the representation of configuration when gateway configuration is set.
routingViaHost: true vs routingViaHost: false